app/models/company.rb

```diff
@@ -51,6 +52,9 @@ class Company < ApplicationRecord
   validates_presence_of :name, :slug, :isin, :market_cap_group
   validates_uniqueness_of :slug, :name
 
+  scope :active, -> { where(active: true) }
+  scope :published_active, -> { where(active: true, visibility_status: 'published') }
```

Comment:
We don't really need that scope. We can write `Company.published.active`

Reply:
I see that my comment goes against @kowal  ;]. But I'm not sure what's the point of having extra scopes if we can combine scopes.

Reply:
I'd keep it simpler and separate scopes as well.